x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/action/saml/TransportSamlLogoutAction.java

```diff
@@ -112,7 +112,7 @@ private SamlLogoutResponse buildResponse(Authentication authentication, Map<Stri
         final String session = getMetadataString(tokenMetadata, SamlRealm.TOKEN_METADATA_SESSION);
         final LogoutRequest logout = realm.buildLogoutRequest(nameId.asXml(), session);
         if (logout == null) {
-            return new SamlLogoutResponse(null);
+            return new SamlLogoutResponse((String)null);
```

Comment:
nit: space

Reply:
I'm going to leave this as is because are already inconsistent about this throughout the codebase.